infra/vpc/vpc-variables.tf

```diff
@@ -0,0 +1,20 @@
+variable "vpc-cidr-block" {
+  description = "VPC CIDR Block"
+  type = string
+  default = "172.16.0.0/16"
+}
+
+variable "vpc-azs" {
```

Comment:
You want to find out which az's are available in the region instead of statical input.